platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/XMLHttpRequest.js

```diff
@@ -87,7 +87,34 @@ export default class XMLHttpRequest extends EventTarget {
   }
 
   send(data = '') {
-    console.error("XMLHttpRequest invalid, my.tb.request does not support http!");
+    if (this.readyState !== XMLHttpRequest.OPENED) {
+      throw new Error("Failed to execute 'send' on 'XMLHttpRequest': The object's state must be OPENED.")
+    } else {
+      let myRequestTask = my.tb.request({
+        body: JSON.stringify(data),
```

Comment:
is data typeof object, it seems data is typeof string
why we use JSON.stringify here